front/src/applications/operationalStudies/views/Project.tsx

```diff
@@ -68,7 +67,7 @@ export default function Project() {
   const [imageUrl, setImageUrl] = useState('');
   const dispatch = useDispatch();
   const navigate = useNavigate();
-  const projectId = useSelector(getProjectID);
+  const { projectId } = useParams();
```

Comment:
I think it's not a good thing to cast it to number and directly use it without any test. Maybe we can do this :
```suggestion
  const { projectId: urlProjectId } = useParams();
  const projectId = useMemo(() => Number(urlProjectId) || INVALID_ID, [urlProjectId]); // set INVALID_ID = -1 outside the component
```